src/components/markdown/markdown.tsx

```diff
@@ -15,13 +14,33 @@ import remarkRaw from 'rehype-raw';
 import rehype2react from 'rehype-react';
 import { isValidElementType } from 'react-is';
```

Comment:
<img width="482" alt="Screen Shot 2021-11-18 at 3 36 52 PM" src="https://user-images.githubusercontent.com/1738349/142500746-f98c3c41-35c2-48bd-81d4-66999f4882b7.png">

This works based on usage but I wonder if we should define this as a function that can take anything and returns a boolean?

Reply:
Added `@types/react-is`